example/TrueSDKSample/HostViewController.swift

```diff
@@ -173,11 +173,35 @@ class HostViewController: UIViewController, UITableViewDataSource, UITableViewDe
         userDataHeightConstraint.constant = CGFloat(userDataModel.count) * propertyRowHeight
         
         TCTrueSDK.sharedManager().delegate = self
+        TCTrueSDK.sharedManager().titleType = .default
+        changeTitleTypeOnNotification()
     }
     
     override func viewWillLayoutSubviews() {
         avatarImageView.layer.cornerRadius = avatarImageView.layer.bounds.height / 2
     }
+    
+    private func changeTitleTypeOnNotification() {
+        NotificationCenter.default.addObserver(forName: .titleChanged, object: nil, queue: nil) { [weak self] (notification) in
+            self?.setTilteType(from: notification)
+        }
+    }
+    
+    private func setTilteType(from notification: Notification) {
+        guard let userInfo = notification.userInfo as? [String: Any],
+            let titleType = userInfo["titleType"] as? TitleType else {
+            return
+        }
+        
+        TCTrueSDK.sharedManager().titleType = titleType
+    }
+    
+    @IBAction func openTitleSelection() {
+        let controller = QATitleSelectionTableViewController()
+        controller.title = "Select Title"
+        let navigationControlelr = UINavigationController(rootViewController: controller)
+        present(navigationControlelr, animated: true, completion: nil)
```

Comment:
if you are presenting the viewcontroller from here why not use delegation why do you use a broadcast from the notification center? it is only observed in one place

Reply:
Done